src/Workspaces/CSharp/Portable/Utilities/FormattingRangeHelper.cs

```diff
@@ -288,6 +288,8 @@ private static SyntaxToken GetPreviousTokenIfNotFirstTokenInTree(SyntaxToken tok
 
         public static bool AreTwoTokensOnSameLine(SyntaxToken token1, SyntaxToken token2)
         {
+            if (token1 == token2) return true;
```

Comment:
nit: could you write this as:

```c#
if (token1 == token2)
{
    return true;
}
```

Tnx! :)

Reply:
Thanks @CyrusNajmabadi for the formatting style suggestion. Thanks @genlu for addressing this, I applied the suggested change.